test/testReference.ts

```diff
@@ -29,12 +29,13 @@
 ///<reference path="components/plots/linePlotTests.ts" />
 ///<reference path="components/plots/areaPlotTests.ts" />
 ///<reference path="components/plots/barPlotTests.ts" />
+///<reference path="components/plots/clusteredBarPlotTests.ts" />
```

Comment:
I think the ordering is in compile order (barPlot must go before things that extend from barPlot)